src/stat.h

```diff
@@ -119,18 +119,23 @@ class Stat1 {
     if (numsamples_ == 0) return VType();
     VType mean = sum_ * (1.0 / numsamples_);
     if (stddev) {
-      VType avg_squares = sum_squares_ * (1.0 / numsamples_);
-      *stddev = Sqrt(avg_squares - Sqr(mean));
+      // Sample standard deviation is undefined for n = 1
+      if (numsamples_ == 1) {
+        *stddev = VType();
+      } else {
+        VType avg_squares = sum_squares_ * (1.0 / numsamples_);
+        *stddev = Sqrt(numsamples_ / (numsamples_ - 1.0) * (avg_squares - Sqr(mean)));
+      }
     }
     return mean;
   }
 
   // Return the standard deviation of the sample set
   VType StdDev() const {
-    if (numsamples_ == 0) return VType();
+    if (numsamples_ <= 1) return VType();
     VType mean = Mean();
     VType avg_squares = sum_squares_ * (1.0 / numsamples_);
-    return Sqrt(avg_squares - Sqr(mean));
+    return Sqrt(numsamples_ / (numsamples_ - 1.0) * (avg_squares - Sqr(mean)));
```

Comment:
And probably we need to use `Mean(&stddev);` for the second line, since `mean` is an unused variable and the Makefile converts warnings into errors.